simapp/cmd/rcd/commands.go

```diff
@@ -81,18 +86,47 @@ func initAppConfig() (string, interface{}) {
 
 	customAppConfig := CustomAppConfig{
 		Config: *srvCfg,
+		Celestia: &relayer.CelestiaConfig{
+			RpcURL:             "https://rpc-mocha.pops.one:443", // TODO remove hardcoded URL
+			RpcTimeout:         30 * time.Second,
+			GasPrice:           "0.01utia",
+			GasAdjustment:      1.0,
+			ProofQueryInterval: 12 * time.Second,
+			MaxFlushSize:       32,
+		},
 	}
 
-	customAppTemplate := serverconfig.DefaultConfigTemplate
+	customAppTemplate := serverconfig.DefaultConfigTemplate + `
+
+	[celestia]
```

Comment:
Should the tiablob default config template live in the module instead of simapp?

Reply:
yes, ty.